PWGCF/FEMTOSCOPY/FemtoDream/AliFemtoDreamTrackCuts.cxx

```diff
@@ -12,122 +12,122 @@
 ClassImp(AliFemtoDreamTrackCuts)
 
 AliFemtoDreamTrackCuts::AliFemtoDreamTrackCuts()
-    : fMCHists(nullptr),
-      fHists(nullptr),
-      fMinimalBooking(false),
-      fMCData(false),
-      fDCAPlots(false),
-      fTOFM(false),
-      fDoMultBinning(false),
-      fCheckMother(false),
-      fCombSigma(false),
-      fContribSplitting(false),
-      fFillQALater(false),
-      fCheckFilterBit(false),
-      fCheckESDFiltering(false),
-      fCheckPileUpITS(false),
-      fCheckPileUpSPDTOF(false),
-      fCheckPileUpTOF(false),
-      fCheckPileUp(false),
-      fFilterBit(0),
-      fpTmin(0.),
-      fpTmax(0.),
-      fpTexmin(0.),
-      fpTexmax(0.),
-      fcutPt(false),
-      fexclPt(false),
-      fetamin(0.),
-      fetamax(0.),
-      fcutEta(false),
-      fcutCharge(false),
-      fCharge(0),
-      fnTPCCls(0),
-      fcutnTPCCls(false),
-      fMaxSharedClsTPC(160),
-      fCutSharedClsTPC(false),
-      fCutChi2(false),
-      fMinCutChi2(0.f),
-      fMaxCutChi2(0.f),
-      fDCAProp(false),
-      fDCAToVertexXY(0),
-      fCutDCAToVtxXY(false),
-      fDCAToVertexZ(0),
-      fCutDCAToVtxZ(false),
-      fCutSharedCls(false),
-      fCheckTPCRefit(false),
-      fCutTPCCrossedRows(false),
-      fCrossedRows(70),
-      fRatioCrossedRows(0.83),
-      fCutPID(false),
-      fAllowITSonly(false),
-      fCutHighPtSig(false),
-      fParticleID(AliPID::kUnknown),
-      fNSigValue(3.),
-      fNSigValueITS(3.),
-      fPIDPTPCThreshold(0),
-      fRejectPions(false) {
+  : fMCHists(nullptr),
+    fHists(nullptr),
+    fMinimalBooking(false),
+    fMCData(false),
+    fDCAPlots(false),
+    fTOFM(false),
+    fDoMultBinning(false),
+    fCheckMother(false),
+    fCombSigma(false),
+    fContribSplitting(false),
+    fFillQALater(false),
+    fCheckFilterBit(false),
+    fCheckESDFiltering(false),
+    fCheckPileUpITS(false),
+    fCheckPileUpSPDTOF(false),
+    fCheckPileUpTOF(false),
+    fCheckPileUp(false),
+    fFilterBit(0),
+    fpTmin(0.),
+    fpTmax(0.),
+    fpTexmin(0.),
+    fpTexmax(0.),
+    fcutPt(false),
+    fexclPt(false),
+    fetamin(0.),
+    fetamax(0.),
+    fcutEta(false),
+    fcutCharge(false),
+    fCharge(0),
+    fnTPCCls(0),
+    fcutnTPCCls(false),
+    fMaxSharedClsTPC(160),
+    fCutSharedClsTPC(false),
+    fCutChi2(false),
+    fMinCutChi2(0.f),
+    fMaxCutChi2(0.f),
+    fDCAProp(false),
+    fDCAToVertexXY(0),
+    fCutDCAToVtxXY(false),
+    fDCAToVertexZ(0),
+    fCutDCAToVtxZ(false),
+    fCutSharedCls(false),
+    fCheckTPCRefit(false),
+    fCutTPCCrossedRows(false),
+    fCrossedRows(70),
+    fRatioCrossedRows(0.83),
+    fCutPID(false),
+    fAllowITSonly(false),
+    fCutHighPtSig(false),
+    fParticleID(AliPID::kUnknown),
+    fNSigValue(3.),
+    fNSigValueITS(3.),
+    fPIDPTPCThreshold(0),
+    fRejectPions(false) {
```

Comment:
Please don't reformat the whole class. Use e.g. this style file: https://github.com/bhohlweger/GentleFemto/blob/master/eclipse-cpp-google-style.xml